src/3bc_macros.h

```diff
@@ -27,6 +27,10 @@
 #define _3BC_MOS6502
 #endif
 
+#if defined(__nuttx__)
```

Comment:
Is there any other compilation constants besides the one defined by the programmer? (such as from the nuttx kernel itself)